recipes/relion/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "RELION" %}
+{% set version = "3.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/3dem/relion/archive/refs/tags/{{ version }}.tar.gz
+  sha256: f10934058e0670807b7004899d7b5103c21ad77841219352c66a33f98586d42e
+
+build:
+  number: 0
+  skip: true  # [win or cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - cmake
+    - make
+  host:
+    - {{ mpi }}
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - libtiff
+    - libpng
+    - fltk
+    - fftw
+  run:
+    - {{ mpi }}
+    - python
```

Comment:
One last question. Do we really need `python`? I think CMake is only install binary executables and the shared library, so I am wondering if this is actually a runtime dependency or not.

Reply:
The only python code I can find in their repo is two scripts in https://github.com/3dem/relion/tree/master/scripts, but they are not being installed to PREFIX. Is this intended or an oversight?

Reply:
you're right - let's remove, there are some optional external dependencies which use Python but let's leave that for now